packages/ds-medicare-gov/src/styles/_base.scss

```diff
@@ -115,3 +115,13 @@ svg {
   line-height: 20px;
   margin-bottom: $spacer-1;
 }
+
+/**
+ * Pagination overrides
+ */
+
+.ds-c-pagination {
+  .ds-c-button {
+    font-family: var(--font-family-heading);
+  }
+}
```

Comment:
Is there a way to do this without CSS overrides? We spent years trying to get CSS overrides _out_ of the Medicare theme and rely on tokens instead.

Reply:
Looks good now 👍